test/format/format.sh

```diff
@@ -123,6 +125,13 @@ while :; do
 	-R)
 		timing_stress_split_test=1
 		shift ;;
+        -r)
+		live_record_binary="$2"
+		if ! [ $(command -v "$live_record_binary") ]; then
```

Comment:
I always put the negation operator inside the test brackets.  I.e:

`if [ ! $(command -v "$live_record_binary") ]; then`

The behavior is the same either way, but a quick search of other scripts in the repo shows this is our usual style.

Reply:
3deaf74